include/sdf/Element.hh

```diff
@@ -112,6 +112,14 @@ namespace sdf
     /// \sa Element::SetRequired
     public: const std::string &GetRequired() const;
 
+    /// \brief Set if the element and children wer set or default in the file
```

Comment:
nit: "wer" -> "were".

Reply:
Fixed in 52cd85f7c78a366d72b266a842c372c48f1588ce